CMakeLists.txt

```diff
@@ -172,9 +172,23 @@ endif()
 find_package(PythonInterp)
 if(PYTHONINTERP_FOUND)
   enable_testing()
-  add_test(NAME iwyu_tests
-    COMMAND ${PYTHON_EXECUTABLE} run_iwyu_tests.py -- $<TARGET_FILE:include-what-you-use>
-    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
+
+  function(ADD_IWYU_TEST NAME FILE)
+    add_test(NAME ${NAME}
+      COMMAND ${PYTHON_EXECUTABLE} run_iwyu_tests.py --run-test-file=${FILE} -- $<TARGET_FILE:include-what-you-use>
+      WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
```suggestion
      WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
```

Reply:
Hi @zaufi, could you please clarify why `SOURCE_DIR` should be replaced with `BINARY_DIR` here?
The expanded command should be something like:
`$ python3 run_iwyu_tests.py ...`
and `run_iwyu_tests.py` is in project's root source directory. I expect `BINARY_DIR` to be a directory where build artifacts are stored (the output iwyu binary).

In order for python interpreter to "see" `run_iwyu_tests.py` the working dir should be the root source dir.

Reply:
Yeah, that seems surprising to me too. `run_iwyu_tests.py` is definitely in the source dir.